src/assets/legwear/leggings/leggings.asset.ts

```diff
@@ -0,0 +1,51 @@
+DefineAsset({
+	name: 'Leggings',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		leggings: {
+			name: 'Leggings',
+			default: '#2A2A2A',
+		},
+	},
+	// size:800, y:574, centered
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Clothing',
+			'Legwear',
+		],
+	},
+	modules: {
+		design: {
+			type: 'typed',
+			name: 'Leggings design',
+			variants: [
+				{
+					id: 'netshowing',
+					name: 'Net showing',
+					default: true,
+				},
+				{
+					id: 'plain',
```

Comment:
Another thing I noticed, where I was not sure if intended or not:

The plain setting only covers the gap on the front, but not the small one on the back.

Reply:
Good catch. Somehow the 'fill' image got broken during export.